### ISSUE TITLE
Update Tests to test Dual Screen Support in Multiple Languages

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#697]
It might also be worth adding a "Test" button in the Advanced Preferences, for end-user troubleshooting purposes?

**Comment by randomeizer** [comment]
Would the Test button email the results to us?

**Comment by latenitefilms** [comment]
Yes, that would be the plan. Maybe the label should be something like "Send Test Results to Developer" or something?

**Comment by latenitefilms** [comment]
@randomeizer - Now that you've done all the locale changes - does this mean this issue can be closed?

**Comment by randomeizer** [comment]
I’ve updated the tests for multiple languages, but haven’t actually added
any tests for dual screens yet. Should do that...

**Comment by latenitefilms** [comment]
@randomeizer - I wonder if we can also run tests for multiple versions of FCPX? That way we could just run a test suite overnight that checks FCPX 10.3.4 and 10.4.3?
